packages/client/src/devices/devices.ts

```diff
@@ -197,7 +174,20 @@ export const getAudioStream = async (
       ...trackConstraints,
     },
   };
-  return getStream(constraints);
+
+  try {
+    await getAudioBrowserPermission().prompt({
+      throwOnNotAllowed: true,
+      forcePrompt: true,
```

Comment:
Enabling camera/microphone is an important enough operation to allow prompting for the permission again if the prompt was previously dismissed.